src/rates/staticRates.ts

```diff
@@ -6,9 +6,13 @@ import { Trade, TradeService } from "./tradeService";
 
 const logger = getLogger();
 
-export type ConfigRates = {
-  [buyAsset in Asset]: { [sellAsset in Asset]?: number };
-};
+interface SubRate {
```

Comment:
🔧 This introduces another interface without a clear purpose and an ambiguous name (from looking at this this it is not immediately clear for me what it is for). Was this added as preparation for the `ERC20` implementation?

Reply:
I could not find a way to do it otherwise

Reply:
Hm... I see. This is because we changed it to be `buy.sell = value` 😅
For now: OK for me if it stays like you implemented it. 

At some point we should revise how to actually depict our `rates` and try to do it in a (standard) format as used by others (e.g. exchanges). 
That was also a point raised by Toby in a discussion today: At the moment it is confusing to people looking at our implementation because we don't stick to terms/formats used in other trading applications.

Reply:
I think it'd be good to have concrete examples for us to know how to make it better.